packages/webex/src/contact-center.js

```diff
@@ -17,4 +18,4 @@ Webex.init = function init(attrs = {}) {
   return new Webex(attrs);
 };
 
-module.exports = Webex;
+export default Webex;
```

Comment:
Have we tested this in node?

Just curious how `require` and `export` live in the same file.